Guide PL [skip ci]

bizley opened the pull request:
- REST error handling,
- structure entry scripts, 
- test overview,
- input validation,
- output client scripts

Commit:
REST error handling, structure entry scripts and test overview

Yupik committed:
input-validation.md

Yupik committed:
output-client-scripts.md 100%

README.md translation fix
output-client-scripts.md 100%
input-validation typo fix

Commit:
Merge branch 'master' of git://github.com/yiisoft/yii2 into guide-pl

SilverFire commented on docs/guide-pl/input-validation.md:
```diff
@@ -0,0 +1,648 @@
+
+## Deklaracja zasad <span id="declaring-rules"></span>
+
+Aby metoda `validate()` naprawdę zadziałała, powinieneś zadeklarować zasady walidacji dla atrybutów które mają jej podlegać.
```
`validate()` -> [[yii\base\Model::validate()|validate()]]

SilverFire commented on docs/guide-pl/input-validation.md:
```diff
@@ -0,0 +1,648 @@
+## Deklaracja zasad <span id="declaring-rules"></span>
+
+Aby metoda `validate()` naprawdę zadziałała, powinieneś zadeklarować zasady walidacji dla atrybutów które mają jej podlegać.
+Powinno zostać to zrobione przez nadpisanie metody [[yii\base\Model::rules()]]. Poniższy przykład pokazuje jak zostały zadeklarowane zasady walidacji dla modelu `ContactForm`:
```
[[yii\base\Model::rules()]] -> [[yii\base\Model::rules()|rules()]]

SilverFire commented on docs/guide-pl/input-validation.md:
```diff
@@ -0,0 +1,648 @@
+
+Jednak z głownych zasad mówi, że nigdy nie powinno ufać się danym otrzymanym od użytkowników oraz zawsze je walidować przez ich użyciem.
+
+Mamy [model](structure-models.md) wypełniony danymi od użytkownika. Możemy go zwalidować przez wywołanie metody [[yii\base\Model::validate()]].
```
[[yii\base\Model::validate()|validate()]]

SilverFire commented on docs/guide-pl/input-validation.md:
```diff
@@ -0,0 +1,648 @@
+Zasada może zostać użyta dla konkretnych [scenariuszy](structure-models.md#scenarios) przez określenie opcji `on`.
+Jeśli nie określisz opcji `on` oznacza to, że zasada zostanie użyta w każdym scenariuszu.
+
+Kiedy zostaje wywołana metoda `validate()` zostają wykonane następujące kroki w celu wykonania walidacji:
```
[[yii\base\Model::validate()|validate()]]

SilverFire commented on docs/guide-pl/input-validation.md:
```diff
@@ -0,0 +1,648 @@
+
+Kiedy zostaje wywołana metoda `validate()` zostają wykonane następujące kroki w celu wykonania walidacji:
+
+1. Określenie które atrybuty powinny zostać zwalidowane przez pobranie listy atrybutów z [[yii\base\Model::scenarios()]] używając aktualnego [[yii\base\Model::scenario|scenario]].
```
[[yii\base\Model::scenarios()|scenarios()]]

SilverFire commented on docs/guide-pl/input-validation.md:
```diff
@@ -0,0 +1,648 @@
+
+1. Określenie które atrybuty powinny zostać zwalidowane przez pobranie listy atrybutów z [[yii\base\Model::scenarios()]] używając aktualnego [[yii\base\Model::scenario|scenario]].
+    Wybrane atrybuty nazywane są *atrybutami aktywnymi*.
+2. Określenie które zasady walidacji powinny zostać użyte przez pobranie listy zasad z [[yii\base\Model::rules()]] używając aktualnego [[yii\base\Model::scenario|scenario]].
```
[[yii\base\Model::rules()|rules()]]

SilverFire commented on docs/guide-pl/input-validation.md:
```diff
@@ -0,0 +1,648 @@
+
+### Zdarzenia walidacji <span id="validation-events"></span>
+
+Podczas wywołania metody [[yii\base\Model::validate()]], zostaną wywołane dwie metody, które możesz nadpisać, aby dostosować proces walidacji:
```
[[yii\base\Model::validate()|validate()]]

SilverFire commented:
Thank you very much for your effort.

I've posted notices for links to related methods. It's not, critical but if the context is about some class, we prefer to make link text shorter. Could you change do the same in your PR, please?

I don't know Polish, so I can't check spelling. @Yupik

Commit:
Guide PL fixes and links shortening

Commit:
Merge branch 'master' of git://github.com/yiisoft/yii2 into guide-pl

bizley commented:
I've made few corrections and shortened methods' links.
With next pull the rest of already translated PL doc will get all links shortened.

SilverFire commented on docs/guide-pl/input-validation.md:
```diff
@@ -0,0 +1,649 @@
+        $statuses = json_encode(Status::find()->select('id')->asArray()->column());
+        $message = json_encode($this->message, JSON_UNESCAPED_SLASHES | JSON_UNESCAPED_UNICODE);
+        return <<<JS
+if ($.inArray(value, $statuses) > -1) {
```
Recently there was a PR that fixed this expression: https://github.com/yiisoft/yii2/commit/83d72cddc02da030a384e87295721ea6a2ce2937

Commit:
Fix

Adding the fix from
https://github.com/yiisoft/yii2/commit/83d72cddc02da030a384e87295721ea6a2ce2937

Yupik committed:
Merge branch 'master' of git://github.com/yiisoft/yii2 into guide-pl

Yupik committed:
Merge branch 'guide-pl' of https://github.com/bizley/yii2 into guide-pl

bizley commented:
Feel free to merge. No ETA on links shortening now.

SilverFire commented:
Thank you very much for your work!
